Dockerfile

```diff
@@ -19,14 +19,14 @@ RUN apk upgrade \
 ARG GITHUB_SHA="dev"
 ENV COMMIT_SHA ${GITHUB_SHA}
```

Comment:
```suggestion
ENV COMMIT_SHA=${GITHUB_SHA}
```

```
- LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 20)
```